runelite-client/src/main/java/net/runelite/client/plugins/microbot/util/settings/Rs2SpellBookSettings.java

```diff
@@ -202,7 +202,6 @@ public static boolean toggleIconResizing(boolean toggle) {
      * @return
      */
     public static boolean setAllFiltersOn() {
-        return Microbot.getClientThread().runOnClientThread(() -> {
```

Comment:
Why did you remove this ?

Reply:
Because it didn't work, it just failed and didn't execute the code properly whenever I tried to use it